include/verilated_vcd_c.h

```diff
@@ -332,6 +318,54 @@ class VerilatedVcd {
         }
     }
 
+    //=========================================================================
+    // Inside dumping routines used by Verilator
+
+    vluint32_t* oldp(vluint32_t code) { return m_sigs_oldvalp + code; }
```

Comment:
I'd suggest removing this function and using m_sings_oldvalp[code] directly when needed.  Compiling without optimization is a reasonable thing to do for compile speed, and removing this extra function would help the runtime speed a lot.

Reply:
`m_signs_oldvalp` is private, and that function is called once per trace activity set (i.e.: once in the prologue per `TRACE_CHANGE_SUB` function), do you think it would really help that much compared to all the actual `chg*` and `full*` being compiled without optimization?

Reply:
Fair point, leave then, thanks.